### DISCUSSION TITLE
What do people use this for?  What's the point?

### DISCUSSION THREAD

**Discussion by PRProd** [PRProd/HA-Firemote#333] [General]
I get this question every once in a while in forums.  Here are a few responses that have been posted
<br>

User [Ok-Jury5684](https://www.reddit.com/user/Ok-Jury5684/) [posted on Reddit](https://www.reddit.com/r/homeassistant/comments/xrje84/comment/iqgy9gz/?utm_source=share&utm_medium=web2x&context=3):
> Here are my usecases:
> - my physical remote is out of possession of my toddlers
> - if I'm watching tv in the dark, I definitely know where my phone is, unlike the remote
> - no need in batteries (well, it's minor, but still).
<br>

User [jdsmn21](https://www.reddit.com/user/jdsmn21/) [posted on Reddit](https://www.reddit.com/r/homeassistant/comments/19eol16/comment/kjhjxmy/?utm_source=share&utm_medium=web2x&context=3):
> Here’s my most recent use case: I’m laying in bed, but I’m hearing machine guns from the boy playing Call of Duty in the basement.
>
>Two screen taps, and I’m looking at a virtual remote to turn his tv volume down.
<br>

User **Pops3351** commented on [buymeacoffee.com](https://www.buymeacoffee.com/prprod/c/8781970):
> Finding Firemote has not only made my day, but has completed my quest to automate everything in my theater and throughout my home. More importantly, it has made it possible for my wife to navigate the Apple TV seamlessly within HA. My trip down the Home Assistant Rabbit Hole is complete. Thanks for sharing your work!
<br>

User @redxtech posted in PR #410 
>I'm setting up the home automation for my family, and I think it's super cool that you can pull up a visually identical remote on your phone, where all the buttons work the same (mostly, still waiting for home_hold).
>
>Plus, when I showed it to my Dad, he thought it was pretty cool as well!
<br>

<br>
So now I ask all of you: What do you use it for?

**Comment by thos-edison** [comment]
**Here is what I use it for...**

I have an onn. device plugged into an inexpensive ($30) video capture device. The capture device allows me to have TV in a resizable window on my PC - that way, if I'm just sitting here doing some menial work or surfing the web, I can have something showing down in the corner. In addition, it has a pass-through, so I can stream to the TV, the computer, or both at the same time - keeping the physical remote handy somewhere else and the Firemote always available in a small windows next to the "TV" on the PC. The TV doesn't always have to be on, so being able to turn the onn. device on/off with Firemote saves steps. It also comes in handy with my "genuine" _Google TV_ device on another TV, in another room. I really want to try it with a DirecTV Stream (AT&T C71KW-400 "Osprey" device, but haven't had the time...we'll see. 😉 

(On a side note: I also have a fax modem connected to HA which allows me to show Caller ID on the onn.  using Notifications for Android TV - which is something you usually don't see with / on a streaming device)

**Comment by PRProd** [comment]
Other than using it while I build it, here is my main use case:

Passwords! 

Because I use a password manager, I have unique, long, and complex passwords for all of my accounts.  Typing something like **$CiK1*AHiy0reUJE0VVPat1uPoEZ"** into a password field using a physical remote is exhausting when you have to log into an app.  Who ever types something like that correctly the first time anyway?

With Firemote, one tap on the remote's keyboard button, and one password paste later, all done!  My accounts are secure, and I'm not annoyed!  :smile:

**Comment by PRProd** [comment]
YouTube user [Un Friki Domotizado](https://www.youtube.com/@Unfrikidomotizado) / GitHub user @Albert267 (I think) made a video demonstrating why he uses it!

Watch the first minute here: [https://www.youtube.com/watch?v=Kp689zsvfFA](https://www.youtube.com/watch?v=Kp689zsvfFA)

**Comment by PRProd** [comment]
The [Self-Hosted](https://selfhosted.show/) podcast mentioned Firemote and how it has helped them in [Episode 135: Rebuilding For the Last Time](https://selfhosted.show/135)

Jump to [5:22 - 7:50](https://selfhosted.show/135?t=322) to hear all about it!

[![image](https://github.com/user-attachments/assets/786468f4-768e-4730-a4cc-4bc47a0e909f)](https://selfhosted.show/135?t=322)

They highlighted things such as ease of use, customization, and the always important Wife Approval Factor!

Thanks for the shoutout!
